src/templates/vibso_classes.tsv

```diff
@@ -46,4 +46,4 @@ obo:VIBSO_0000042	spectral resolution interval as energy		https://github.com/NFD
 obo:VIBSO_0000043	radiometric resolution interval		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/125	A resolution interval that quantifies the minimum difference in irradiated energy, often coded as gray level in a grayscale or look-up-table (LUT).	ISBN 081946043-5	owl:Class	resolution interval	ORCID:0000-0003-2673-366X|ORCID:0000-0003-2673-366X|ORCID:0000-0002-2239-3955			This class is actually to general for VIBSO's scope. It needs to be discussed with CHMO and OBI devs which of these is a better home, as there currently does not exist an optical device ontology within OBO Foundry. OBI contains most terms around optical devices so far. 			
 obo:VIBSO_0000044	temporal resolution interval		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/125	A resolution interval that quantifies the temporal separation between two measurements or frames, often expressed as the inverse of the frame rate. In techniques where temporal resolution is not directly derived from frame rate, it quantifies the minimum temporal interval between two resolvable events.	ISBN 081946043-5	owl:Class	resolution interval	ORCID:0000-0003-2673-366X|ORCID:0000-0003-2673-366X|ORCID:0000-0002-2239-3955		In techniques where the temporal resolution is not given directly from a temporal distance between subsequent measurements (e.g., pulsed time resolved pump-probe techniques), it can be used more properly as defined above	This class is actually to general for VIBSO's scope. It needs to be discussed with CHMO and OBI devs which of these is a better home, as there currently does not exist an optical device ontology within OBO Foundry. OBI contains most terms around optical devices so far. 			
 obo:VIBSO_0000045	diffraction limit		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/125	A spatial resolution interval that quantifies the theoretical maximum spatial resolution achievable by an optical system, constrained by the diffraction of light. 	ISO/DIS 18115-3, https://www.iso.org/obp/ui#d25805417e256	owl:Class	spatial resolution interval	ORCID:0000-0003-2673-366X|ORCID:0000-0003-2673-366X|ORCID:0000-0002-2239-3955		The Abbe diffraction limit, is defined as λ/ (2 NA) in which λ is the wavelength of the illuminating light and NA is the numerical aperture.	This class is actually to general for VIBSO's scope. It needs to be discussed with CHMO and OBI devs which of these is a better home, as there currently does not exist an optical device ontology within OBO Foundry. OBI contains most terms around optical devices so far. 			
-											
\ No newline at end of file
+obo:......	spectral window center		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/88	A setting datum which determines the middle wavelength/wavenumber of a spectrum and corresponds to the centering of the spectrometer. It is the average of the spectral window range maximum and minimum.		owl:Class		ORCID:0000-0002-1595-3213|ORCID:0000-0002-2239-3955				
```

Comment:
My proposal to rewrite to:
A setting datum that specifies the average of the maximum and minimum of the spectral window range utilized during the acquisition of spectroscopic data, and that corresponds to the optimal alignment or centering of the spectrometer to ensure that the most relevant spectral features are captured effectively within the defined acquisition range.

We should also add an example similar to the minimum and maximum.

Reply:
```suggestion
obo:......	spectral window center		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/88	A setting datum that specifies the middle wavelength/wavenumber of a spectrum, equal to the average of the maximum and minimum of the spectral window range utilized during the acquisition of spectroscopic data. It corresponds to the optimal alignment or centering of the spectrometer to ensure that the most relevant spectral features are captured effectively within the defined acquisition range.		owl:Class		ORCID:0000-0002-1595-3213|ORCID:0000-0002-2239-3955				
```